### ISSUE TITLE
Postcss-reporter not displaying line numbers

### ISSUE THREAD

**Issue by renatoruk** [postcss/postcss-reporter#22]
I have set up gulp postcss with stylelint plugin and I want to display warnings in console. Current postcss plugin setup consists of stylelint with post-scss syntax succeeding with postcss-reporter.

The problem is, post-css reporter is formatting the console output, but somehow line numbers are not outputed, although in not-formatted output line numbers are shown immediately after absoulte path.

![output](https://cloud.githubusercontent.com/assets/12981417/13203704/ec69528e-d8bd-11e5-95e9-306bcbc5444c.JPG)

This is the code for the lint task, the problem stays if pass object with `clearMessages: true`

```
gulp.task('lintSCSS', function() {
    return gulp.src( 'static/scss/pages/*.scss'
                     // 'static/scss/modules/*.scss',
                     // 'static/scss/helpers/*.scss'
                    )
        .pipe($.postcss([
                    stylelint(styleLintConfig),
                    reporter(),
                        ],

                    { syntax: postscss }
                    
                    ))
```

Can you please tell me is it the issue with my setup or piping stream or something else?
Using Windows 10 Pro 64bit if it may be of any help.

**Comment by davidtheclark** [comment]
I'm thinking there's a very good chance this is a Windows-specific problem, because I haven't seen it before. If you wouldn't mind taking some time at some point to help debug, that would be really awesome (since I don't have a Windows machine) and it should actually be pretty easy. You should be able to open `node_modules/postcss-reporter/lib/formatter.js`, try changing things around or logging extra stuff, and run the task again.

**Comment by renatoruk** [comment]
Changing this line in util.js fixed the problem.

`  var messageInput = _.get(messageNode, 'source.start');`

changed to `'source.start'` from `'source.input'` 

![source](https://cloud.githubusercontent.com/assets/12981417/13204989/35c12d70-d8dd-11e5-80a6-ad6fafc02b4d.JPG)

Plugin returns object with source property and looks for the line number in input property, which in my enviroment contains only inputed scss. 

Could this change be applied to the whole plugin or other preprocessors such as less depend on `'source.input'` property?

**Comment by davidtheclark** [comment]
Thanks! That wasn't a Windows problem at all, but a regression I introduced. Sorry. Fix should be released in 1.3.3

**Comment by renatoruk** [comment]
Great! Thank you!
